lib/components/conversation/conversation_item.css

```diff
@@ -0,0 +1,90 @@
+.conversation-item {
+  padding: 12px;
+  border-left: 1px solid #ccc;
```

Comment:
can we extract the colours into variables? I think it would help to ensure consistency between borders, text colour, status etc. It might make sense to do it in a follow up PR and cover the other css files as well

Reply:
Definitely, will do in a follow up PR if that is okay!